samples/notebooks/noise.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using QDK to simulate Pauli noise\n",
+    "This notebook shows how to run simulations with Pauli noise, such as bit-flip or depolarizing noise.\n",
+    "\n",
+    "First, make sure prerequisites are available. Packages `qsharp` and `qsharp_widgets` must be already installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import qsharp\n",
+    "import qsharp_widgets\n",
+    "%pip install pandas"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulation with noise\n",
+    "\n",
+    "Define a simple program that creates a Bell state on two qubits and measures both qubits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 238,
+   "metadata": {
+    "vscode": {
+     "languageId": "qsharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%qsharp\n",
+    "\n",
+    "operation BellPair() : Result[] {\n",
+    "    use q = Qubit[2];\n",
+    "    H(q[0]);\n",
+    "    CNOT(q[0], q[1]);\n",
+    "    MResetEachZ(q)\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run 20 shots without noise and display results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = qsharp.run(\"BellPair()\", 20)\n",
+    "results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that measurements always agree within a shot es expected. Now run 20 shots of the same program with 10% [depolarizing noise](https://en.wikipedia.org/wiki/Quantum_depolarizing_channel). Depolarizing noise is applied to each gate and each measurement."
```

Comment:
Fixed typo.
I think 20 shots is better here because all results are shown as a list. 100 or 1000 will be overwhelming. This is a simple example where user can look at the actual results rather than some aggregation.

Reply:
How do you decide when to show and list and when to show a histogram? Is there a reason not to just stick with one output format?

Reply:
I want to start with an example that is as simple as possible. This example only runs simulation of a very simple program with a simple noise - nothing else. As a result, user sees pure output of the program. I believe this helps user understand what's going on. Once user sees how to run simulations with noise and how noise affects output, a bit more complex programs and processing gets introduced.